### ISSUE TITLE
[Gunbad] Broken Loot/Quest

### ISSUE THREAD

**Issue by VharneIzor12** [WarEmu/WarBugs#14071]
Issues should be unique. Check if someone else reported 
the issue first, and please don't report duplicates.
If its ToK related, please check the wiki before posting: 
https://wiki.returnofreckoning.com/index.php/Main_Page

**Expected behavior and actual behavior:**

**Steps to reproduce the problem:**

**Screenshots/Videos or archive.org evidences:**

**Comment by VharneIzor12** [comment]
Was running Mount Gunbad and got to Glomp The Squigmasta, Died at around 60(ish) percent and the fight continued. The tank kited the boss around the room to the opposite side from where I was standing and finally killed him after nine minutes. Once the boss died I received a message stating the screenshot attached here.
![Vharne_001](https://user-images.githubusercontent.com/50536408/57577514-67379100-743e-11e9-97ad-6c714008cd99.jpg)
After which I was ressed and was given the normal window for a roll on loot.
The two Redeye Shoulders dropped as shown in the screenshot below. I rolled need for the ironmantle and Stingz rolled greed as shown here as well. I contacted a GM asking for assistance, but to directed here as GMs can not fix loot. The pictures were taken after the conversation with the GM (wrathchild)
![Vharne_000](https://user-images.githubusercontent.com/50536408/57577517-79b1ca80-743e-11e9-9baa-1762d46f17d5.jpg)
This screenshot is to show where both Jaura and Stingz saw the incident happen in real time and gave permission to use them as witnesses as well as the conversation with the GM.
![Vharne_003](https://user-images.githubusercontent.com/50536408/57577533-edec6e00-743e-11e9-9971-4edaf23527b3.jpg)
